proposals/0024/GOVERNANCE.md

```diff
@@ -102,8 +103,135 @@ The following apply to the repo or the part of the codebase for which one would
+
+**Defined by:** _maintainers_ entry in an OWNERS file in a repo owned by the Tinkerbell project.
+Maintainer status is scoped to the repo/code in the Tinkerbell org.
+
+### Requirements
+
+Maintainers of a repository are typically limited to a relatively small group of decision makers and updated as fits the needs of the repository.
+
+The following apply to the repository for which one would be an owner.
+
+- Deep understanding of the technical goals and direction of the repository
+- Deep understanding of the technical domain of the repository
+- Sustained contributions to design and direction by doing all of:
+  - Authoring and reviewing proposals
+  - Initiating, contributing and resolving discussions (emails, GitHub issues, meetings)
+  - Identifying subtle or complex issues in designs and implementation PRs
+- Directly contributed to the repository through implementation and / or review
+
+### Responsibilities and Privileges
+
+The following apply to the repository for which one would be an owner.
+
+- Make and approve technical design decisions for the repository.
+- Set technical direction and priorities for the repository.
+- Define milestones and releases.
+- Mentor and guide Approvers, reviewers, and contributors to the repository.
+- Ensure continued health of repository
+  - Adequate test coverage to confidently release
+  - Tests are passing reliably (i.e. not flaky) and are fixed when they fail
+- Ensure a healthy process for discussion and decision making is in place.
+- Work with other repository owners to maintain the project's overall health and success holistically
+
+___
+
+## Admin
+
+Admins are the GitHub Administrators for the Tinkerbell project.
+
+**Defined by:** _owner_ role in the Tinkerbell GitHub organization.
+
+### Requirements
+
+This team is responsible for holding Org Owner privileges over the Tinkerbell org.
+
+- Nominations to this team will come from the maintainers and require confirmation from the maintainers and Approvers before taking effect.
+- Maintain active membership in the Tinkerbell community.
+
+### Responsibilities and Privileges
+
+Admins are responsible for:
+
+- Administrative tasks in the Tinkerbell Github organization
+
+___
+
+## Inactive members
+
+>This inactive members section follows the guideline defined in the Kubernetes community<sup>[5]</sup>
+
+_Members are continuously active contributors in the community._
+
+A core principle in maintaining a healthy community is encouraging active
+participation. It is inevitable that people's focuses will change over time and
+they are not expected to be actively contributing forever.
+
+However, being a member of the Tinkerbell GitHub organization comes with
+an [elevated set of permissions]. These capabilities should not be used by those
+that are not familiar with the current state of the Tinkerbell project.
+
+Therefore members with an extended period away from the project with no activity
```

Comment:
Could be worth adding a boomerang clause. Life happens and people might have to step away, but given certain timeframes, we could always add them back

Reply:
I think the statement provided in the PR works as a boomerang if we change the two occurrences of `will` to `may`.